script.sinaweibo/resources/lib/utilities/sina_tweet.py

```diff
@@ -0,0 +1,179 @@
+
+import datetime
+import time
+import os
+import xbmcgui
+import xbmc
+import json
+from twitter import *
+from common_addon import *
+from addonfileio import FileIO
+from pyweibo import APIClient
+import traceback
+import urllib2
+import re
+
+
+APP_KEY = "3726251905"
+APP_SECRET = "6673ae2beb1d836af4f9142802fa4357"
+CALLBACK_URL = 'https://api.weibo.com/oauth2/default.html'
+client = APIClient(app_key=APP_KEY, app_secret=APP_SECRET, redirect_uri=CALLBACK_URL)
+client.set_access_token('2.006ctQRCL2yKEE588ff1b69a06mbKW', 1525805999)
+
+
+con_secret = 'h7hSFO1BNKzyB2EabYf7RXXd5'
```

Comment:
YOU ARE NOT ALLOWED TO USE OTHER USERS KEY. This key belongs to matchcenter.

Reply:
Matchcenter is a great add-on which I borrowed a lot from, thank you. Sorry for the misused key trouble.